roxie/ccd/ccdmain.cpp

```diff
@@ -1129,10 +1132,56 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
                 if (port)
                 {
                     const char *protocol = roxieFarm.queryProp("@protocol");
+                    const char *passPhrase = nullptr;
+                    const char *certFile = nullptr;
+                    const char *keyFile = nullptr;
+                    if (protocol && streq(protocol, "ssl"))
+                    {
+#ifdef _USE_OPENSSL
+                        certFile = roxieFarm.queryProp("@certificateFileName");
+                        if (!certFile)
+                        {
+                            WARNLOG("Skipping Roxie SSL Farm Listener on port %d due to missing certificateFileName tag", port);
```

Comment:
I don't really have strong feelings either way - should be consistent with roxie behaviour on other configuration problems...